src/pages/EarningsInvoices.vue

```diff
@@ -0,0 +1,341 @@
+          </td>
+          <td>
+            {{ presentDate(invoice.date_created) }}
+          </td>
+          <td>
+            {{ presentDate(invoice.date_due) }}
+          </td>
+          <td>
+            {{ invoice.id }}
+          </td>
+          <td class='amount-cell'>
+            {{ presentHolofuelAmount(invoice.amount) }}
+          </td>
+          <td>
+            {{ invoice.payment_status }}
+          </td>
+          <td>
+            {{ invoice.exception_status }}
+          </td>
+        </tr>
+      </table>
+    </div>
+    <div class='footer'>
+      Rows per page:&nbsp;&nbsp;
+      <select v-model="pageSize" class="dropdown">
+        <option v-for="option in pageSizeOptions" :value="option" :key="option">
+          {{ option }}
+        </option>
+      </select>
+      <div class='pagination'>
+        {{ paginationLegend }}
+        <RightChevronIcon @click="hasPrevPage && goToPrevPage()" class="page-arrow-left" :color="hasPrevPage ? '#00CAD9' : '#606C8B'" />
+        <RightChevronIcon @click="hasNextPage && goToNextPage()" class="page-arrow-right" :color="hasNextPage ? '#00CAD9' : '#606C8B'" />
+      </div>
+    </div>
+  </PrimaryLayout>
+</template>
+
+<script>
+
+import PrimaryLayout from 'components/PrimaryLayout.vue'
+import ShortUpArrowIcon from 'components/icons/ShortUpArrowIcon.vue'
+import RightChevronIcon from 'components/icons/RightChevronIcon.vue'
+import mockInvoiceData, { PSTATUS_LATE, PSTATUS_PAID, PSTATUS_UNPAID, ESTATUS_EXCEPTION } from 'src/mockInvoiceData'
+import { presentPublisherHash, presentHolofuelAmount } from 'src/utils'
+
+const SORT_HAPP = 'hApp'
+const SORT_PUBLISHER = 'Publisher'
+const SORT_CREATED = 'Date Created'
+const SORT_DUE = 'Date Due'
+const SORT_INVOICE = 'Invoice #'
+const SORT_AMOUNT = 'Amount'
+const SORT_PAYMENT = 'Payment Status'
+const SORT_EXCEPTION_STATUS = 'Exception Status'
+
+const FILTER_ALL = 'All'
+const FILTER_UNPAID_LATE = 'Unpaid & Late'
+const FILTER_PAID = 'Paid'
+const FILTER_EXCEPTIONS = 'Exceptions'
+
+export default {
+  name: 'EarningsInvoices',
+  components: {
+    PrimaryLayout,
+    ShortUpArrowIcon,
+    RightChevronIcon
+  },
+  data () {
+    return {
+      invoices: mockInvoiceData,
+      headers: [
+        SORT_HAPP,
+        SORT_PUBLISHER,
+        SORT_CREATED,
+        SORT_DUE,
+        SORT_INVOICE,
+        SORT_AMOUNT,
+        SORT_PAYMENT,
+        SORT_EXCEPTION_STATUS
+      ],
+      filter: FILTER_ALL,
+      sort: SORT_CREATED,
+      sortDesc: true,
+      pageSize: 10,
+      page: 0
+    }
+  },
+  created () {
+    this.filterOptions = [FILTER_ALL, FILTER_UNPAID_LATE, FILTER_PAID, FILTER_EXCEPTIONS]
+    this.pageSizeOptions = [5, 10, 20, 30, 50]
+
+    const queryFilter = this.$route.query.filter
+    if (queryFilter === 'unpaid') {
+      this.filter = FILTER_UNPAID_LATE
+    } else if (queryFilter === 'exceptions') {
+      this.filter = FILTER_EXCEPTIONS
+    }
```

Comment:
Are there supposed to be branches for the other two filters?

Reply:
no, because there are no direct links to the other filters. We can add those branches if we ended up needing to link directly to them.